app/controllers/admin/listing_shapes_controller.rb

```diff
@@ -139,22 +172,23 @@ def uneditable_fields(process_summary)
   end
 
   def render_new_form(form, process_summary, available_locs)
-    locals = common_locals(form, process_summary, available_locs)
+    locals = common_locals(form, 0, process_summary, available_locs)
     render("new", locals: locals)
   end
 
-  def render_edit_form(id, form, process_summary, available_locs)
-    locals = common_locals(form, process_summary, available_locs).merge(
+  def render_edit_form(id, form, count, process_summary, available_locs)
+    locals = common_locals(form, count, process_summary, available_locs).merge(
       id: id,
       name: pick_translation(form[:name])
     )
     render("edit", locals: locals)
   end
 
-  def common_locals(form, process_summary, available_locs)
+  def common_locals(form, count, process_summary, available_locs)
```

Comment:
actually, it needs to be there since the code would throw undefined local var without this.